### PR TITLE
test: discover also python files which don't contain hash-bang when running pylint

### PR DESCRIPTION
**Pull request by KKoukiou**
And fix all the errors that appear.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,7 +20,7 @@ WEBLATE_REPO=tmp/weblate-repo
 WEBLATE_REPO_URL=https://github.com/cockpit-project/cockpit-machines-weblate.git
 WEBLATE_REPO_BRANCH=main
 
-PYEXEFILES=$(shell git grep -lI '^#!.*python')
+PYEXEFILES=$(shell git grep -lI '^#!.*python') $(shell find test/ -name *.py ! -path 'test/common/*')
```
Comment (martinpitt) [inline Makefile:23]: Seems a bit weird to call these `PYEXEFILES`. Either call these `PYTHONFILES` and use `git ls-tree -r --name-only HEAD | grep '\.py'` (to really only catch the tracked files, and avoid doing this `! -path` stuff) or call pyflakes/pycodestyle without arguments to let them find all the py files (not 100% sure that this works).

Comment (skobyda) [inline Makefile:23]: I would also extend the find command to src/ directory. There is for example a file `src/lib/inotify.py` which is not covered by either command

**Review by skobyda — CHANGES_REQUESTED** [review]

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks!

File: Makefile (hunk 1 above)
Comment (martinpitt) [reply Makefile:23]: Indeed, my `ls-tree` proposal from below finds these as well.

**Review by skobyda — APPROVED** [review]

**Review by martinpitt — APPROVED** [review]
:+1:

**Pull request merged**